CHANGES.md

```diff
@@ -2,6 +2,14 @@
 
 ### 1.72 - 2020-08-03
 
+##### Breaking Changes :mega:
+
+- CesiumJS no longer ships with a default Mapbox access token and Mapbox imagery layers have been removed from the `BaseLayerPicker` defaults. `accessToken` is now required when constructing `MapboxImageryProvider` or `MapboxStyleImageryProvider`. This only affects out-of-the-box behavior and applications using their own Mapbox access token are unaffected.
```

Comment:
- Probably link to documentation for `MapboxApi`, which includes instructions for how to create an access token
- Do you think it's worth having sample code for how to add it back to the `BaseLayerPicker`?

Reply:
1. Since anyone already using the Mapbox layers would have already done this, I'm not sure there's a reason to do so.

2. These were always for demonstration only, users always had to add their own if the plan was to use them in their own application since the BaseLayerPicker defaults includes multiple layers that are for demo purpose only (ESRI, OSM, ion). There is definitely a larger CesiumJS issue where the defaults for BaseLayerPicker lead the user down a bad path as far as usage goes, but that is being handled in a different thread.